### DISCUSSION TITLE
Unable to run the seed command

### DISCUSSION THREAD

**Discussion by Mic75** [tada5hi/typeorm-extension#377] [Q&A]
I try to run the seed command in the context of a NestJS project.
The command is:
`node -r dotenv/config ./node_modules/typeorm-extension/dist/cli/index.js seed -r dist/src -d ormconfig`

I am preloading (node -r) typeorm-extension env variables from a .env thanks to dotenv. Which contains:
```
TYPEORM_SEEDING_FACTORIES=./modules/data-access/factories/**/*{.ts,.js}
TYPEORM_SEEDING_SEEDS=./modules/data-access/seeds/**/*{.ts,.js}
```

In the built project (under dist folder), the data-source file is located at: 
`./dist/src/ormconfig.js`

Seeds and factory are respectively located under:
`./dist/src/modules/data-access/seeds/`
and
`./dist/src/modules/data-access/factories/`

When running the seed command I am getting the following error:
```
MissingDriverError: Wrong driver: "undefined" given. Supported drivers are: "aurora-mysql", "aurora-postgres", "better-sqlite3", "capacitor", "cockroachdb", "cordova", "expo", "mariadb", "mongodb", "mssql", "mysql", "nativescript", "oracle", "postgres", "react-native", "sap", "sqlite", "sqljs", "spanner".
```

Which does not really makes sense.
It's not clear to me what I am doing wrong, any help would be greatly appreciated.

**Comment by Mic75** [comment]
I finally managed to have it running, by replacing `node` by `ts-node` with the following env vars:
```
TYPEORM_SEEDING_FACTORIES=./dist/src/modules/data-access/factories/**/*{.ts,.js}
TYPEORM_SEEDING_SEEDS=./dist/src/modules/data-access/seeds/**/*{.ts,.js}
```

The error message was misleading, nothing to do with the db driver, but rather the DataSource being instantiated with an invalid object as params.
 
While  debugging, it appears that typeorm-extension is doing erroneous assumptions when running the cli with node and a tsconfig.json is present in the current working directory.

I can setup a basic repo @tada5hi for reproducing the issue if you'd like to.

**Reply by tada5hi** [reply]
@Mic75 Glad to hear, that you managed to get it working :innocent: .
The error regarding the missing driver, is indead pretty strange and it would be realy great if you could create a basic repo :v: :blush: 

Greetings
Peter

**Reply by Mic75** [reply]
Thanks for your answer. 
Sure thing, will do a basic repo to repro the issue.

Thanks again for this great extension and taking over the compatibility with typeorm 0.3.*. Its really useful.
Cheers.

**Reply by tada5hi** [reply]
Thank you.

You are very welcome. Glad you like it :blush:
